README.md

```diff
@@ -213,6 +213,22 @@ ReverseProxy /pboard/ http://dashes.acme:5000/pboard/
 ProxyPassReverse /pboard/ http://dashes.acme:5000/pboard/
 ```
 
+Using the puppetlabs/apache module:
+```puppet
```

Comment:
can you please add a newline between the next and the clodeblock? Otherwise a markdown linter complains